PrivateCollections/Germany/Neukirchen003.xml

```diff
@@ -0,0 +1,1145 @@
+                             Data on leaves dimensions
+                             taken from <locus target="#27r"></locus>.
+                           </note>
+                        </extent>
+                        <foliation>
+                          <list>
+                            <item xml:id="fol1" resp="DE">
+                              <locus from="1r" to="138v"></locus>
+                              Foliation in pencil in the right bottom corner of the folio. This is the foliation used as reference in this record.
+                            </item>
+                          </list>
+                        </foliation>
+                        <condition key="good">Stains of water and dirt throughout the manuscript. First two quires have been
+                        reattached to the binding with a green thread. The textblock has been reattached to the binding. The leather
+                      cover of the binding is partially cracked, rubbed off and damaged by insects.</condition>
+                         </supportDesc>
+                     <layoutDesc>
+                           <layout corresp="#ms_i1.1 #ms_i1.2 #ms_i1.3" columns="1" writtenLines="24">
+                             <locus from="3r" to="124v"></locus>
+                             <ab type="ruling">Ruling and pricking are visible.</ab>
+                           </layout>
+                           <layout corresp="#ms_i1.4" columns="2" writtenLines="25">
+                             <ab type="ruling">Ruling and pricking are visible.</ab>
+                           </layout>
+
+                     </layoutDesc>
+
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script"/>
+                        <seg type="ink">Black, red</seg>
+                        <seg type="rubrication">First one or two lines in the beginning of the text and text sections; numerals; punctuation marks;
+                        name of God, Mary, and angels; single letters and words.</seg>
+                        <desc/>
+                     </handNote>
+                  </handDesc>
+                     <decoDesc>
+                       <decoNote type="miniature" xml:id="d1">
+                         <locus target="#2v"></locus>
+                         <desc>A miniature of the <ref type="authFile" corresp="AT1066DavidHarp"></ref> in which <persName ref="PRS3417David">David</persName> is
+                         playing his harp in front of the <persName ref="PRS10775Trinity">Holy Trinity</persName> and
+                         <persName ref="PRS12016Evangelists">four Evangelists</persName> depicted as Four Beasts with inscriptions
+                         <foreign xml:lang="gez">ገጸ፡ ንስር፡ ገጸ፡እንስሳ፡ ገጸ፡<gap reason="illegible"></gap> ገጸ፡ አንበሳ፡</foreign>. The miniature is colored in yellowish brown colour.</desc>
```

Comment:
color but coloured :) I believe officially we follow British English

Reply:
done